docs/admin/kubernetes_deployment.md

```diff
@@ -1,52 +1,55 @@
 
-Kubernetes 1.22 is the minimally required version and please check the following recommended Istio versions for the corresponding
-Kubernetes version.
-
-## Recommended Version Matrix
-| Kubernetes Version | Recommended Istio Version |
-|:-------------------|:--------------------------|
-| 1.27               | 1.18, 1.19                |
-| 1.28               | 1.19, 1.20                |
-| 1.29               | 1.20, 1.21                |
-
-## 1. Install Istio 
-
-The minimally required Istio version is 1.13 and you can refer to the [Istio install guide](https://istio.io/latest/docs/setup/install).
-
-Once Istio is installed, create `IngressClass` resource for istio.
-```yaml
-apiVersion: networking.k8s.io/v1
-kind: IngressClass
-metadata:
-  name: istio
-spec:
-  controller: istio.io/ingress-controller
-```
-
-
-!!! note 
-    Istio ingress is recommended, but you can choose to install with other [Ingress controllers](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/) and create `IngressClass` resource for your Ingress option.
-
-
-
-## 2. Install Cert Manager
+## 1. Install Cert Manager
 The minimally required Cert Manager version is 1.9.0 and you can refer to [Cert Manager installation guide](https://cert-manager.io/docs/installation/).
 
 !!! note
     Cert manager is required to provision webhook certs for production grade installation, alternatively you can run self signed certs generation script.
 
-## 3. Install KServe
+## 2. Install KServe
 !!! note 
-    The default KServe deployment mode is `Serverless` which depends on Knative. The following step changes the default deployment mode to `RawDeployment` before installing KServe.
-
+    The default KServe deployment mode is `Serverless` which depends on Knative. The following step changes the default deployment mode to `RawDeployment` before installing KServe and the `RawDeployment` mode does not have any dependecy on Knative.
 
-**i. Install KServe**
+**i. Create a yaml file**
+Open an editor of your choice and create `kustomization.yaml` file.     
+    ```yaml
+    
+    apiVersion: kustomize.config.k8s.io/v1beta1
+    kind: Kustomization
+
+    namespace: kserve
+
+    resources:
+    - https://github.com/kserve/kserve/releases/download/v0.13.0/kserve.yaml
+
+    patches:
+    - target:
+        kind: ConfigMap
+        name: inferenceservice-config
+      patch: |
+        - path: /data/deploy
+          op: replace
+          value: |-
+            {
+                "defaultDeploymentMode": "RawDeployment"
+            }
+        - path: /data/ingress
+          op: replace
+          value: |-
+            {
+                "disableIstioVirtualHost": true,
+                "disableIngressCreation": true,
+                "ingressDomain": "svc.cluster.local",
+                "ingressGateway" : "disabled",
+                "ingressService" : "disabled"
```

Comment:
here we also need to set the default ingressClass.

Reply:
Please suggest what change needs to be made to the `yaml` file.

Reply:
@spolti Can you provide some suggested verbiage here for @Rajakavitha1?

Reply:
it is described a few lines below, one of the removed.

btw @terrytangyuan, shouldn't we have a specific section for raw?
Instead of deleting the istio part, we should instead document both ways. 
Istio is required for serverless and it is the default method.

You could change the beginning of the doc like keeping the requirements for the Serverless mode, and later, explain what is the `RawDeployment` (like it is done now) and describe the steps needed to install it.

Reply:
```
                "disableIstioVirtualHost": true,
                "disableIngressCreation": true,
                "ingressGateway" : "disabled",
                "ingressService" : "disabled"
```

I have a few questions regarding this configuration.

Isn't it true that the controller already handles these aspects? In other words, when using rawDeployment, these related calls may not even be triggered. Should we really be making changes in this area?
```
                "ingressDomain": "svc.cluster.local",
```
this would be needed.

```
"defaultDeploymentMode": "RawDeployment"
```
Additionally, I understand that this setting implies using rawDeployment as the default, but it doesn't mean that only rawDeployment will be used. I think this configuration might still be necessary when setting the serverlessMode in isvc.

Lastly, shouldn't the disableIngressCreation be set to false instead? does the ingress need not to be created?